tests/src/Functional/EventIntegrationTest.php

```diff
@@ -101,6 +101,7 @@ public function testCreateEvent(): void {
     $page->fillField('Title', 'Example Event title');
     $page->fillField('Content', 'Example Event content');
     $page->fillField('Introduction', 'Example Event introduction');
+    $page->fillField('Registration URL', 'https://europa.eu');
```

Comment:
testing the field is definitely needed but we also need a separate test for the formatter, independent, since the formatter will move to whitelabel, check how we did it there (it needs to go into the helper module)